src/Views/AppListUpdateView.vala

```diff
@@ -157,18 +157,25 @@ namespace AppCenter.Views {
                 uint update_numbers = 0U;
                 uint nag_numbers = 0U;
                 uint64 update_real_size = 0ULL;
+                bool _check_flatpak = true;
+                bool using_flatpak = false;
                 foreach (var package in get_packages ()) {
                     if (package.update_available || package.is_updating) {
                         if (package.should_nag_update) {
                             nag_numbers++;
                         }
 
+                        if (_check_flatpak && package.is_flatpak) {
```

Comment:
You don't need the extra `_check_flatpak` variable here, you can just use `!using_flatpak`

Reply:
Oh duh, you're smart.